pkg/kubectl/describe.go

```diff
@@ -2526,13 +2528,18 @@ func (fn typeFunc) Describe(exact interface{}, extra ...interface{}) (string, er
 	return s, err
 }
 
+// printLabelsMultilineWithFilter prints filtered multiple labels with a proper alignment.
+func printLabelsMultilineWithFilter(out io.Writer, title string, labels map[string]string, skipMap map[string]bool) {
+	printLabelsMultilineWithIndent(out, "", title, "\t", labels, skipMap)
+}
+
 // printLabelsMultiline prints multiple labels with a proper alignment.
 func printLabelsMultiline(out io.Writer, title string, labels map[string]string) {
-	printLabelsMultilineWithIndent(out, "", title, "\t", labels)
+	printLabelsMultilineWithIndent(out, "", title, "\t", labels, nil)
 }
 
 // printLabelsMultiline prints multiple labels with a user-defined alignment.
-func printLabelsMultilineWithIndent(out io.Writer, initialIndent, title, innerIndent string, labels map[string]string) {
+func printLabelsMultilineWithIndent(out io.Writer, initialIndent, title, innerIndent string, labels map[string]string, skipMap map[string]bool) {
```

Comment:
The name of this function is terrible since it isn't just about labels.  Fine for now though.